packages/gatsby/src/bootstrap/page-hot-reloader.js

```diff
@@ -45,7 +46,7 @@ const runCreatePages = async () => {
     })
     .map(p => p.id)
 
-  const timestamp = new Date().toJSON()
+  const timestamp = Date.now()
```

Comment:
`new Date().toJSON()` => "2019-02-20T10:47:41.468Z"
`Date.now()` => 1550659670910

I don't know how page.updatedAt looks so no idea if this is good or bad 😄

Reply:
Yeah... for some reason I made a few types (like page) use unix timestamps instead of iso8601 :-(

So this check was failing before.